model-api-gen/src/main/kotlin/org/modelix/metamodel/generator/TypescriptMMGenerator.kt

```diff
@@ -68,6 +68,13 @@ class TypescriptMMGenerator(val outputDir: Path, val nameConfig: NameConfig = Na
             """
             }}
             }
+            ${if (!includeTBarrels) {
+                ""
+            } else {
```

Comment:
The old function name `generateRegistry` does not reflect the new code.
Rename to `generateIndexTs`.

Reply:
Fixed.